v2/apiserver/internal/core/events.go

```diff
@@ -702,18 +718,12 @@ type EventsStore interface {
 	// parameter in the underlying data store to reflect that they have been
 	// canceled. Implementations MUST only cancel events whose Workers have not
 	// already reached a terminal state.
```

Comment:
Can you amend this comment to reflect that implementations MUST return a count of affected Events?